### PR TITLE
feat(view-pager): swipeEnabled prop

### LINKED ISSUE #1210: [ViewPager] - Disable swipe gestures
## 🚀 Feature Proposal

I would like to disable ViewPager swipe gestures.

## Motivation

Im trying to implement a multistep form using the component.

## Example

I have a multistep form, and each step need to be validated before proceeding, I would like to disable the swipe gestures so I can guarantee that the user will not jump any steps.

### PR DESCRIPTION
**Pull request by petruspierre**
### Please read and mark the following check list before creating a pull request:

 - [X] I read and followed the [CONTRIBUTING.md](https://github.com/akveo/react-native-ui-kitten/blob/master/CONTRIBUTING.md) guide.
 - [X] I read and followed the [New Feature Checklist](https://github.com/akveo/react-native-ui-kitten/blob/master/DEV_DOCS.md) guide.

 #### Short description of what this resolves:
 
 Closes #1210

### CODE REVIEW

File: src/components/ui/viewPager/viewPager.component.tsx [hunk 1]
```diff
@@ -191,14 +195,20 @@ export class ViewPager<ChildrenProps = {}> extends React.Component<ViewPagerProp
...
   public render(): React.ReactElement<ViewProps> {
-    const { style, children, ...viewProps } = this.props;
+    const { style, children, swipeEnabled, ...viewProps } = this.props;
+
+    const animatedViewProps = swipeEnabled ? {
+      ...viewProps,
+      ...this.panResponder.panHandlers,
+    } : {
+      ...viewProps,
+    };
```
Comment (artyorsh) [inline src/components/ui/viewPager/viewPager.component.tsx:205]: Let's simplify a little
```suggestion
        const panResponderConfig = swipeEnabled ? this.panResponder.panHandlers : null;
        const animatedViewProps = { ... viewProps, ...panResponderConfig  };
```

**Review by artyorsh — COMMENTED** [review]
Thanks for tackling this! 👍 
Can we apply some code suggestions before it got merged?

**Review by artyorsh — APPROVED** [review]

**Comment by artyorsh** [comment]
🎉

**Comment by bencreynolds** [comment]
Is it possible to pass this property to TabBar? I'm trying to put a Top TabBar inside of a modal (react-native-modalize) but having horizontal swiping is anti-pattern behavior. I'm reading the code and the TabView component contains ViewPager but I don't believe there's a way to pass props. I could be wrong, I'm very new to trying to understand/extend functionality to a 3rd party library.

Any help/input is appreciated.

**Pull request merged**